presto-product-tests/README.md

```diff
@@ -0,0 +1,37 @@
+# Presto product tests
+
+## Test configuration
+
+Test assume that you have hadoop and presto cluster running. For setting up development environment we
```

Comment:
Could you describe your motivation for why you dislike launching product tests from Maven. Why is launching integration tests OK and product tests not? Currently we can launch product tests using both IDE (e.g IntelliJ) and Maven. Launching product tests from IntellJ is extremely useful when testing Presto in local development environment. But some people prefer to start tests from command line, therefore we would prefer to keep Maven way of launching product tests too.

However, as you mentioned, we could additionally make Maven to build an executable product tests JAR with all dependencies (that would probably also include Presto) that could be easily launched on test cluster. That would require a maven hack to produce such a jar but maybe that will be better than launching tests directly from maven in some cases.